src/darwin/Framework/CHIP/MTRDeviceControllerDataStore.mm

```diff
@@ -151,6 +151,17 @@ - (nullable instancetype)initWithController:(MTRDeviceController *)controller
     return self;
 }
 
+- (void)startUpWithClusterDataHandler:(MTRDeviceControllerDataStoreClusterDataHandler)clusterDataHandler
```

Comment:
Seems to me like this is more like fetchAttributeDataForAllDevices or something?  This `startUpWithClusterDataHandler` is a bit odd, because we are not really starting up the controller data store...

Reply:
Done.